triangle_facts/triangle_facts.rb

```diff
@@ -1,37 +1,22 @@
-# Killer facts about triangles AWW YEAH
 class Triangle
-	attr_accessor :side1,:side2,:side3
+	attr_accessor :side1, :side2, :side3
 
-	def initialize(side1,side2,side3)
-		@side1,@side2,@side3 = side1,side2,side3
+	def initialize(side1, side2, side3)
+		@side1 = side1
+		@side2 = side2
+		@side3 = side3
 	end
 
-	def equilateral()
-		return side1 == side2 && side2 == side3
+	def equilateral
+		side1 == side2 && side2 == side3
 	end
 
-	def isosceles()
-		return [side1,side2,side3].uniq.length == 2
+	def isosceles
+		[side1, side2, side3].uniq.length == 2
 	end
 
-	def scalene()
-		unless ! (equilateral || isosceles)
-	    false
-	  else
-	    true
-	  end
-	end
-
-	def recite_facts
-		puts 'This triangle is equilateral!' if equilateral
-		puts 'This triangle is isosceles! Also, that word is hard to type.' if isosceles 
-		puts 'This triangle is scalene and mathematically boring.' if scalene 
-
-		angles = self.calculate_angles(side1,side2,side3)
-		puts 'The angles of this triangle are ' + angles.join(',')
-
-		puts 'This triangle is also a right triangle!' if angles.include? 90
-		puts ''
+	def scalene
+		!(equilateral || isosceles)
```

Comment:
Nitpick: I would personally do `!equilateral && !isosceles` because I have a hard time doing "not (or)" logic in my head.